packages/notification-center/src/i18n/languages/sa.ts

```diff
@@ -2,9 +2,9 @@ import { ITranslationEntry } from '../lang';
 
 export const SA: ITranslationEntry = {
   translations: {
-    notifications: 'विज्ञप्तयः',
+    notifications: 'अधिसूचनाः',
     markAllAsRead: 'सर्वं पठितम् इति सूचनीयम्',
-    poweredBy: 'नियन्त्रणकृत्',
+    poweredBy: 'द्वारा संचालितः',
```

Comment:
Are the colons needed? I do not know sanskrit so I don't if it is part of the a sanskrit character or just adding the colons as a punctuation mark.

Reply:
It is the part of the sanskrit language so it needs to be there!